style.css

```diff
@@ -0,0 +1,74 @@
+body {
+  font-family: sans-serif;
+  color: white;
+  background-color: rgb(31, 16, 16);
+  line-height: 1.4;
+  width: 100%;
+}
+
+section header {
+  font-family: 'Times New Roman', Times, serif;
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 30px;
+  letter-spacing: 20px;
+}
+
+#navbar {
+  position: fixed;
+  height: 100%;
+  min-width: 300px;
+  width: 300px;
+  top: 0;
+  left: 0;
+  color: black;
+  background-color: white;
+  padding: 75px 0;
+  z-index: 3;
+}
+
+#navbar header {
+  color: black;
+  margin: 10px;
+  text-align: center;
+  font-size: 1.8em;
+}
+
+#navbar li {
+  list-style: none;
+  padding: 5px 0;
+  position: relative;
+  width: 100%;
+}
+
+#navbar a {
+  color: gray;
+  text-decoration: none;
+}
+
+#main-doc {
+  position: absolute;
+  top: 30vw;
+  margin-left: 310px;
+  padding: 20px;
+}
+
+section {
+  background: rgb(230, 255, 255, 0.1);
+  padding: 2.5rem 2.5rem;
+  border-radius: 2rem;
+  width: 500px;
+  margin: auto;
+}
```

Comment:
It is best if you use use flexbox styling to style the sections properly

Reply:
* Kindly explain why you request a change - it might be obvious for you but not for someone else

Reply:
you did a good job styling the page but there is one or two minor changes required 👍 , It is considered best practice if you use flexbox to layout your page sections, the reason you'd choose it is that you want to lay a collection of items out in one direction or another by simply having a container `section` tag around your different sections and having `display: flex` in the container section's css styling, please consider also centring your sections for a better lay out !